### PR TITLE
WIP: Provide Alpine-based images for imagebuilder and sdk

### PR DESCRIPTION
**Pull request by oxr463**
I wasn't sure how you wanted them named, so I just overwrote the original Dockerfiles.

See: https://github.com/openwrt/docker/issues/5

### CODE REVIEW

**Comment by aparcar** [comment]
Are you sure do offer *only* alpine images? I'd prefer to have both...

**Comment by aparcar** [comment]
> I wasn't sure how you wanted them named, so I just overwrote the original Dockerfiles.

Oops sorry! Well we can make folders like `sdk/Dockerfile.{debian,alpine}`

**Comment by oxr463** [comment]
> Oops sorry! Well we can make folders like sdk/Dockerfile.{debian,alpine}

No worries, we definitely want both. Let me try creating directories next.

**Comment by oxr463** [comment]
Is Circle CI not configured to run on PRs?

**Comment by aparcar** [comment]
No, will enable it but with disabled docker upload/secrets.

File: docker-imagebuilder.sh [hunk 1]
```diff
@@ -6,40 +6,43 @@ set -x
 TARGETS="${TARGETS:-x86-64}"
 BRANCHES="${BRANCHES:-master}"
 DOCKER_IMAGE="${DOCKER_IMAGE:-openwrt-imagebuilder}"
+DISTROS="alpine debian"
```
Comment (aparcar) [inline docker-imagebuilder.sh]: I'd be in favor to only build one of the two locally per default, and the full range only via CI

Comment (aparcar) [reply docker-imagebuilder.sh]: `DISTROS="${DISTROS:-debian}"`

File: docker-imagebuilder.sh [hunk 2]
```diff
@@ -6,40 +6,43 @@ set -x
...
+            grep openwrt-imagebuilder sha256sums > sha256sums_imagebuilder
+            sha256sum -c sha256sums_imagebuilder
+
+            mkdir -p ./imagebuilder
+            tar Jxf $IMAGEBUILDER_FILE --strip=1 -C ./imagebuilder
+            rm -rf $IMAGEBUILDER_FILE
+
+            docker build -t "$DOCKER_IMAGE:$TARGET-$BRANCH" -f imagebuilder/"Dockerfile.${DISTRIBUTION}" .
+
+            rm -rf ./imagebuilder
```
Comment (aparcar) [inline docker-imagebuilder.sh]: This removes the folder containing the ImageBuilder Dockerfiles, as the IB archive is also extracted to this folder. I guess we have to choose different folder names. Same happens with the SDK

**Comment by aparcar** [comment]
Hey are you up to rework this PR? If not I can look into that

**Comment by oxr463** [comment]
I don't think I have time to right now.

**Comment by oxr463** [comment]
@aparcar any progress on this? Otherwise I can go ahead and work on rebasing my work on the latest master. This repository has come pretty far in the last few months!

**Comment by aparcar** [comment]
Please feel free to continue the work. I'm currently working on other projects

**Comment by aparcar** [comment]
@oxr463 Are you still interested in this? If not I'd close this PR

**Comment by oxr463** [comment]
> @oxr463 Are you still interested in this? If not I'd close this PR

I can always pick this up later.

**Comment by lu-zero** [comment]
What was missing to  land it?

**Pull request closed without merging**